supported_examples.md

```diff
@@ -1,220 +1,67 @@
-			<td><a href="https://github.com/huggingface/text-generation-inference">TGI</a></td>
-			<td>Xeon/Gaudi2</td>
-			<td>Language Translation</td>
-		</tr>
-	</tbody>
-</table>
+| Framework                                                                      | LLM                                                           | Serving                                                         | HW          | Description          |
+| ------------------------------------------------------------------------------ | ------------------------------------------------------------- | --------------------------------------------------------------- | ----------- | -------------------- |
+| [LangChain](https://www.langchain.com)/[LlamaIndex](https://www.llamaindex.ai) | [haoranxu/ALMA-13B](https://huggingface.co/haoranxu/ALMA-13B) | [TGI](https://github.com/huggingface/text-generation-inference) | Xeon/Gaudi2 | Language Translation |
 
 ### SearchQnA
 
 [SearchQnA](./SearchQnA/README.md) is an example of chatbot for using search engine to enhance QA quality.
 
-<table>
-	<tbody>
-		<tr>
-			<td>Framework</td>
-			<td>LLM</td>
-			<td>Serving</td>
-			<td>HW</td>
-			<td>Description</td>
-		</tr>
-		<tr>
-			<td><a href="https://www.langchain.com">LangChain</a>/<a href="https://www.llamaindex.ai">LlamaIndex</a></td>
-			<td><a href="https://huggingface.co/Intel/neural-chat-7b-v3-3">NeuralChat-7B</a></td>
-			<td><a href="https://github.com/huggingface/text-generation-inference">TGI</a></td>
-			<td>Xeon/Gaudi2</td>
-			<td>Chatbot</td>
-		</tr>
-		<tr>
-			<td><a href="https://www.langchain.com">LangChain</a>/<a href="https://www.llamaindex.ai">LlamaIndex</a></td>
-			<td><a href="https://huggingface.co/mistralai/Mistral-7B-v0.1">Mistral-7B</a></td>
-			<td><a href="https://github.com/huggingface/text-generation-inference">TGI</a></td>
-			<td>Xeon/Gaudi2</td>
-			<td>Chatbot</td>
-		</tr>
-	</tbody>
-</table>
+| Framework                                                                      | LLM                                                               | Serving                                                         | HW          | Description |
+| ------------------------------------------------------------------------------ | ----------------------------------------------------------------- | --------------------------------------------------------------- | ----------- | ----------- |
+| [LangChain](https://www.langchain.com)/[LlamaIndex](https://www.llamaindex.ai) | [NeuralChat-7B](https://huggingface.co/Intel/neural-chat-7b-v3-3) | [TGI](https://github.com/huggingface/text-generation-inference) | Xeon/Gaudi2 | Chatbot     |
+| [LangChain](https://www.langchain.com)/[LlamaIndex](https://www.llamaindex.ai) | [Mistral-7B](https://huggingface.co/mistralai/Mistral-7B-v0.1)    | [TGI](https://github.com/huggingface/text-generation-inference) | Xeon/Gaudi2 | Chatbot     |
 
 ### VisualQnA
```

Comment:
This may be put in another topic:
Should VisualQnA Table format be aligned with other examples? 
| Framework | LLM | Serving | HW | Description |